configuration/ibm/50001001.json

```diff
@@ -28,7 +28,8 @@
     "frus": {
         "/sys/bus/i2c/drivers/at24/8-0050/eeprom": [
             {
-                "inventoryPath": "/system/chassis/motherboard",
+                "inventoryPath": "/xyz/openbmc_project/inventory/system/chassis/motherboard",
```

Comment:
This is not the latest JSON, please pick latest JSON and make change to that.
Also please push JSON changes in a separate PR so that it can get merged without waiting for read to get it.